nep-28.mediawiki

```diff
@@ -2,7 +2,7 @@ NEP: 28
 Title: Standardization of the Smallest Unit of GAS
 Author: Jimmy Liao <jimmy@r3e.network>
 Type: Standard
-Status: Draft
+Status: Final
```

Comment:
BTW, while we're here, let's fix the formatting. Standard header should be sorrounded by `<pre>` tag with corresponding indentation for proper MD rendering. Here's how does it looks like right now:
![image](https://github.com/user-attachments/assets/e4a57cd5-23b4-4d25-9fc1-afe2baf6c1d8)
And here's how should it be:
![image](https://github.com/user-attachments/assets/be4e8689-cae9-40ec-9cfb-9b4145c16d95)